CHANGELOG.adoc

```diff
@@ -12,33 +12,41 @@ and this project adheres to http://semver.org/spec/v2.0.0.html[Semantic Versioni
 ==== Versions
 toc::[]
 
-== https://github.com/d4l-data4life/hc-util-sdk-kmp/compare/1.8.1...main[Unreleased]
+== https://github.com/d4l-data4life/hc-util-sdk-kmp/compare/1.9.0...main[Unreleased]
+
+=== Added
+
+=== Changed
+
+=== Removed
+
+=== Fixed
+
+=== Bumped
+
+=== Migration
+
+== https://github.com/d4l-data4life/hc-util-sdk-kmp/compare/v1.8.1...v1.9.0[1.9.0]
 
 === Added
 
 * CommonResourceLoad (util-test)
-* HttpMockObjectResponse (util-test)
 * util-coroutine module
 * D4LSDKFlow (util-coroutine)
 * util-coroutine-test module
 * util-ktor-test module
 * HttpMockObjectResponse (util-ktor-test)
 * createHelloWorldOkResponse, createHelloWorldMockClient, createErrorMockClient and createMockClientWithResponse (util-ktor-test)
+* stately (1.1.6)
 
 === Changed
 
 * _**BREAKING**_ runBlockingTest and runWithContextBlockingTest are now part of util-coroutine-test
```

Comment:
these might need some migratino instruction?

Reply:
Not that I know of, since it completely new and only relates to util-coroutine...but the latter is worth to mention

Reply:
done